components/Common/Dropdown/index.module.scss

```diff
@@ -14,17 +14,31 @@
     > button {
       background: none;
       border: none;
-
       color: var(--color-text-primary);
       cursor: pointer;
       font-size: var(--font-size-body1);
       padding: var(--space-12);
       text-align: center;
       width: 100%;
-
-      &:hover {
-        background-color: var(--color-dropdown-hover);
-      }
     }
   }
+
+  &:hover {
```

Comment:
This hover change may have been intentional. I guess you should revert it, as hover should only apply to the current entry you're hovering